extensions/shared/register-jetpack-block.js

```diff
@@ -6,7 +6,7 @@ import { registerBlockType } from '@wordpress/blocks';
 /**
  * Internal dependencies
  */
-import extensionList from '../setup/index.json';
+import extensionList from '../index.json';
```

Comment:
Nice move, like this people will find this right away when they start developing blocks.